File: libraries/hdm/functions/types/response/BackupResponseDTO.java
package hdm.functions.types.response;

import java.io.Serializable;

public class BackupResponseDTO
  implements ResponseDTO, Serializable
{
  private long backupId;

  public long getBackupId()
  {
    return this.backupId;
  }

  public void setBackupId(long backupId)
  {
    this.backupId = backupId;
  }
}

File: libraries/hdm/functions/types/response/BackupResponseDTO.js
/* @org.eclipse.vjet.dsf.resource.utils.CodeGen("VjoGenerator") */
vjo.ctype('hdm.functions.types.response.BackupResponseDTO') //< public
.satisfies('hdm.functions.types.response.ResponseDTO')
.satisfies('java.io.Serializable')
.protos({
    backupId:0, //< private long
    //> public long getBackupId()
    getBackupId:function(){
        return this.backupId;
    },
    //> public void setBackupId(long backupId)
    setBackupId:function(backupId){
        this.backupId=backupId;
    }
})
.endType();